token-service/src/main/java/org/p2p/token/service/repository/configurator/CanBeHiddenConfigurator.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.token.service.repository.configurator
+
+import org.p2p.core.token.MetadataExtension
+import org.p2p.core.token.Token
+import org.p2p.core.token.TokenExtensions
+
+class CanBeHiddenConfigurator(
+    private val extensions: MetadataExtension,
```

Comment:
why fields instead of params to config? 
you can create list of configurators inside root configurator and just call .config on each element

Reply:
cause this task not discovered properly, so for now I just pass full object instead of some fields